### ISSUE TITLE
Will you upload an available train_e2e.py?

### ISSUE THREAD

**Issue by zhiyong1997** [ZhaoJ9014/Multi-Human-Parsing_MHP#21]
Dear Mr. Zhao,

As mentioned in the title, the script train_e2e.py has some syntax errors, such as the missing of commas in line 211, and the inconsistency of 'tabs' and 'space' in line 247. I need a human parsing tool for some downstream application, but these problems as well as the missing of adversarial loss, which is mentioned in the paper, make me unconfident about the result and discourage me from training a model myself. 

 Sincerely looking forward to your modification of the code and released pre-trained model in the future. 

Thank you.
Zhiyong
